<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <widget class="QWidget" name="todas">
          <attribute name="title">
           <string>Todas las asignaturas</string>
          </attribute>
          <widget class="QTableWidget" name="tablaCompleta">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>551</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nota</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cursada</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="complete">
          <attribute name="title">
           <string>Completas</string>
          </attribute>
          <widget class="QTableWidget" name="tablaCom">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>551</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
            <property name="font">
             <font>
              <fontweight>Medium</fontweight>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nota</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cursada</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="cursadas">
          <attribute name="title">
           <string>Cursadas</string>
          </attribute>
          <widget class="QTableWidget" name="tablaCur">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>551</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nota</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cursada</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="pendientes">
          <attribute name="title">
           <string>Pendientes</string>
          </attribute>
          <widget class="QTableWidget" name="tablaPendiente">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>551</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nota</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cursada</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="Convalidadas">
          <attribute name="title">
           <string>Convalidadas</string>
          </attribute>
          <widget class="QTableWidget" name="tablaConvalidada">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>551</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nota</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cursada</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Cursando</string>
          </attribute>
          <widget class="QTableWidget" name="tablaCursando">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>551</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nota</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cursada</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipo</string>
            </property>
           </column>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btIntroducir">
       <property name="text">
        <string>Marcar asignatura como cursando</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Añadir qualificación</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
